nilearn/_utils/docs.py

```diff
@@ -940,12 +940,13 @@ def custom_function(vertices):
 
 # threshold
 docdict["threshold"] = """
-threshold : a number, None, or 'auto', optional
+threshold : :obj:`int` or :obj:`float`, None, or 'auto', optional
     If `None` is given, the image is not thresholded.
-    If a number is given, it is used to threshold the image:
-    values below the threshold (in absolute value) are plotted as transparent.
-    If "auto" is given, the threshold is determined magically
-    by analysis of the image.
+    If number is given, it must be non-negative. The specified value is used to
+    threshold the image: values below the threshold (in absolute value) are
+    plotted as transparent.
+    If "auto" is given, the threshold is determined by analysis of
+    the image.
```

Comment:
I am wondering if we could point somewhere for the curious to actually see how the "auto" thresholding is done?